### ISSUE TITLE
FoxyLinks Feature Request: Select multiple links on a webpage and copy title and/or URL

### ISSUE THREAD

**Issue by Bamajon1974** [erosman/support#614]
Good afternoon

Is FoxyLinks still under development? Last update was three years ago. 

Anyway, is it possible to add a feature where one can select one or more links on a webpage and copy the title and/or URL to the clipboard? And be able to copy the Title and URLs on the same line or different lines?

Thanks!

**Comment by erosman** [comment]
> Is FoxyLinks still under development? 

Sure... it hasn't been updated since there hasn't been any feature request.
I have also been working on other addons.

Once I start on updating an addon, I will check all feature requests.

> Anyway, is it possible to add a feature where one can select one or more links on a webpage and copy the title and/or URL to the clipboard? And be able to copy the Title and URLs on the same line or different lines?

Do you mean title of the link or the page?

**Comment by Bamajon1974** [comment]
Correct. What I had in mind is to select multiple links at once, right click on the selection, and copy the name, URL, name+URL on the same line, and name+URL on separate lines. Does that make sense?
